server/broadcast.go

```diff
@@ -604,3 +630,26 @@ func updateSession(sess *BroadcastSession, res *ReceivedTranscodeResult) *Broadc
 
 	return newSess
 }
+
+func refreshSession(sess *BroadcastSession) (*BroadcastSession, error) {
```

Comment:
Let's make sure to test the error paths in this function. `refreshSession()` can be unit tested on its own to avoid the test setup cruft involved with `transcodeSegment()`.

Reply:
fixed in 20340ceabc68452fb4c8914a568e965e1c5f5d30